src/app/components/privacy-policy/privacy-policy.component.ts

```diff
@@ -132,6 +132,17 @@ export class PrivacyPolicyComponent implements OnInit {
    * Scrolls to the top of the page
    */
   scrollToTop() {
-    this.document.body.scrollTop = this.document.documentElement.scrollTop = 0;
+    // this.document.body.scrollTop = this.document.documentElement.scrollTop = 0;
+    const bodyScrollRate = this.document.body.scrollTop / 100;
+    const eleScrollRate = this.document.documentElement.scrollTop / 100;
+
+    const scrollInt = setInterval(() => {
+      this.document.body.scrollTop -= bodyScrollRate;
+      this.document.documentElement.scrollTop -= eleScrollRate;
+      console.log(' Scrolling Body and Element ');
+      if ( this.document.body.scrollTop === 0 && this.document.documentElement.scrollTop === 0 ) {
+        clearInterval(scrollInt);
```

Comment:
I am not a good fan of using pure javascript in Angular application. Isn't we can do the same using 

```
scroll(el: HTMLElement) {
    el.scrollIntoView({behavior:"smooth"});
  }

```

Also, the same method can be used to scroll to down/up when user will click on contents; written in the left sidebar. WDYT ?

Reply:
have you checked this method to solve the smooth scroll issue?